src/Module/Debug/ItemBody.php

```diff
@@ -11,36 +11,43 @@
 use Friendica\Core\System;
 use Friendica\DI;
 use Friendica\Model\Post;
-use Friendica\Network\HTTPException;
```

Comment:
I've used this convention extensively through the code base where I only import the HTTP exceptions once, otherwise I feel like it clutters the `use` block for little benefit.

Reply:
Cluttering the `use` block is a good thing because it shows you the code smell. If the `use` block is to big its a sign that the class does too much.

Reply:
In general I'd agree, however I think exceptions are the exception. For best results, exceptions should be as unique as possible (although this can also be achieved with error codes). We already are simplifying to only HTTP-like exceptions, but there still are dozens of them, and I don't think using several of them where appropriate is a code smell.

Reply:
Fully agreed. In this class imo there aren't much dependencies. But with every new used class one will get a better feeling if the class starts to getting a monolith.

Another great thing about writing every FQCN in the `use` block is that you can easily find all places where a class is used i.e. searching for `use Friendica\Network\HTTPException\NotFoundException`.